modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-api/bnd.bnd

```diff
@@ -1,6 +1,6 @@
 Bundle-Name: Liferay Headless Commerce Admin Catalog API
 Bundle-SymbolicName: com.liferay.headless.commerce.admin.catalog.api
-Bundle-Version: 25.1.2
+Bundle-Version: 26.0.0
```

Comment:
You cannot include bnd.bnd changes with buildREST. `gw buildREST` regenerates the REST files. bnd changes are done as part of `gw baseline`. You need a separate commit for this with following message "COMMERCE-10088 Semantic Versioning (gw baseline)

Reply:
can you check if it's alright now?